main/src/coordinator.cpp

```diff
@@ -38,6 +38,11 @@ void Coordinator::onStart() {
 
     this->flightRecorderController = new FlightRecorderController(this->environment);
     this->flightRecorderController->enableFlightRecorderWindow();
+
+    ConnectionManager* cm = new ConnectionManager();
+    this->chatViewModel = new ChatViewModel(environment, "chat 1", cm);
+    //this->chatViewModel2 = new ChatViewModel(environment, "chat 2", cm);
```

Comment:
For a issue mentioned in the chat_view_model, I am not going to make a dual connection for now as the callbacks from of the first connection gets overridden by the second. Some more research on socket groups will be needed to make more than ons pipe, managing the callbacks and filtering will also require some consideration as soon as it is established how the rendezvous works in the steam networking lib